services/api/src/routes/__tests__/auth.js

```diff
@@ -144,14 +255,68 @@ describe('/1/auth', () => {
       expect(payload).toHaveProperty('kid', 'user');
       expect(payload).toHaveProperty('type', 'user');
 
-      const updatedUser = await User.findOne({
+      const dbUser = await User.findOne({
+        email,
+      });
+      expect(dbUser.email).toBe(email);
+      expect(dbUser.phoneNumber).toBe(phoneNumber);
+      expect(dbUser.authInfo[0].jit).toBe(payload.jit);
+      expect(dbUser.roles).toBe([]);
+    });
+
+    it('should not allow roles to set', async () => {
```

Comment:
need help here, to figure out how to set the writeScope for roles (which is an array)

Reply:
Let me rephrase, the issue here is that we should **not allow** the roles to be set on registration.
it should be filtered out. 

Right now this test fails as it allow a random user to register as an admin user.

I am not sure how to set the writeScope on roles to prevent that.

Reply:
Ah ok ... I think it may be an issue with the writeScope. I've fixed a ton of issues in the new package so let's let that test fail and after we get everything merged up it should be passing!